wp-content/plugins/wporg-learn/inc/post-meta.php

```diff
@@ -181,6 +181,9 @@ function register_misc_meta() {
 				'default'           => 'en_US',
 				'sanitize_callback' => __NAMESPACE__ . '\sanitize_locale',
 				'show_in_rest'      => true,
+				'auth_callback'     => function() {
+					return current_user_can( 'edit_posts' );
+				},
```

Comment:
This is unrelated, but I believe I should have added it when registering this new field, so improving now.